app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_massive = command.split(" ")
```

Comment:
no need touse indexes here, u could unpack ur vars like this:
`command, file_in, file_out = command.split()`
also there is no need to pass space as arg in split method, it use space by default